cli/src/opts/cast.rs

```diff
@@ -298,6 +301,17 @@ pub enum Subcommands {
         #[structopt(subcommand)]
         command: WalletSubcommands,
     },
+    #[structopt(name = "interface", about = "generate contract's interface from ABI")]
+    Interface {
+        #[structopt(help = "The contract's address")]
+        contract_address: Option<String>,
+        #[structopt(help = "pragma version")]
+        pragma: Option<String>,
```

Comment:
Same idea here, use `#[structopt(default_value = "pragma solidity ^0.8.10")]` and make it `pragma: String` instead of Option, should simplify the consuming code